Octokit/Models/Response/EventInfo.cs

```diff
@@ -204,6 +204,23 @@ public enum EventInfoState
         /// url of the reference's source.
         /// </summary>
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Crossreferenced")]
-        Crossreferenced
+        Crossreferenced,
+
+        /// <summary>
+        /// The issue was reveiewed.
+        /// </summary>
+        Reviewed,
+
+        /// <summary>
+        /// A line comment was made.
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Linecommented")]
+        Linecommented,
```

Comment:
Could these be cased as `LineCommented` and `CommitCommented`? Or is this a deserialization issue?